### PR TITLE
fix: `learn more` button on community templates

### LINKED ISSUE #28218: [Bug]: Community templates UI issues
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

1. Learn more button is not working on the `Showcase` tab for community portal
2. There is a lot of extra space in showcase tab.

### Steps To Reproduce

1. Open any app.
2. Open share modal.
3. Go to `Showcase` tab.

### Public Sample App

_No response_

### Environment

Production

### Issue video log

_No response_

### Version

Appsmith v1.9.41-SNAPSHOT

### PR DESCRIPTION
**Pull request by rahulbarwal**
## Description
Adds link for `learn more` buttton in publish to community template modal.
#### PR fixes following issue(s)
Fixes #28218
> if no issue exists, please create an issue and ask the maintainers about this first
>
>
#### Media
Screenshot 2023-10-19 at 1 16 18 PM

#### Type of change
- Bug fix (non-breaking change which fixes an issue)
- [ ] Manual
- [ ] JUnit
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by akash-codemonk — DISMISSED** [review]

File: app/client/src/pages/Editor/CommunityTemplates/Modals/CommunityTemplatesPublishInfo.tsx [hunk 1]
```diff
@@ -94,7 +94,14 @@ const UnPublishedAppInstructions = ({
         </Text>
       </InfoContainer>
       <InfoFooter>
-        <Button endIcon="link" kind="tertiary" size="md">
+        <Button
+          endIcon="link"
+          href="https://community.appsmith.com/library"
```
Comment (dipyamanbiswas07) [inline app/client/src/pages/Editor/CommunityTemplates/Modals/CommunityTemplatesPublishInfo.tsx:99]: can we add it from a constant file please?

**Comment by dipyamanbiswas07** [comment]
Can we also create an issue and add it in this PR please

File: app/client/src/pages/Editor/CommunityTemplates/Modals/CommunityTemplatesPublishInfo.tsx (hunk 1 above)
Comment (rahulbarwal) [reply app/client/src/pages/Editor/CommunityTemplates/Modals/CommunityTemplatesPublishInfo.tsx]: Done.

**Review by dipyamanbiswas07 — APPROVED** [review]
LGTM

**Pull request merged**